app/actions.py

```diff
@@ -953,9 +951,13 @@ def fileWrite(self):
         self.fileStat = os.stat(self.fullPath)
         self.setMessage('File saved')
       except Exception as e:
-        self.setMessage(
-            'Error writing file. The file did not save properly.',
-            color=3)
+        if self.isReadOnly:
+          self.setMessage("Permission error. Try modifing in sudo mode.",
+                          color=3)
```

Comment:
Try changing these to use app.color, like
```
          self.setMessage("Permission error. Try modifing in sudo mode.",
                          color=app.color.get('error'))
```

Reply:
Done